2Homework25.02.22/StackCalculator/StackCalculator.Tests/StackCalculatorTests.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using NUnit.Framework;
+
+namespace StackCalculator.Tests;
+
+[TestFixture]
+public class StackCalculatorTests
+{
+    private StackCalculator _calculator = new StackCalculator();
```

Comment:
```suggestion
    private StackCalculator _calculator = new();
```
Don't repeat yourself